docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -34,59 +47,109 @@ Optionally, when you do not want to store the token on disk, append the `--show-
 
 ## Logging out
 
-Log out to expire the API ML token and remove it from your base profile.
+Log out to prompt the API ML token to expire and remove it from your base profile.
 
-Issue the following command:
+To log out of the API ML:
 
-```zowe auth logout```
+```
+zowe auth logout
+```
 
-The token is expired. Log in again to obtain a new token.
+The token expires. Log in again to obtain a new token.
```

Comment:
"The token expires" seems a bit awkward. Maybe it could be something like: "This will cause the token to expire"